Makefile

```diff
@@ -0,0 +1,21 @@
+
+.SILENT:
+.PHONY: build-sparkctl clean-build
+
+SPARK_OPERATOR_GOPATH=/go/src/github.com/GoogleCloudPlatform/spark-on-k8s-operator
```

Comment:
Can we use `$GOPATH/src/github.com/GoogleCloudPlatform/spark-on-k8s-operator` instead?

Reply:
sure. also why not uploading the binaries to github releases?